contracts/modules/LoanOpenings.sol

```diff
@@ -375,6 +375,9 @@ contract LoanOpenings is LoanOpeningsEvents, VaultController, InterestUser, Swap
 			sentValues.collateralTokenSent = sentValues
 				.collateralTokenSent /// collateralTokenReceived
 				.add(receivedAmount);
+
+			/// Check the minEntryPrice with the rate
+			require(sentValues.loanTokenSent >= sentValues.minEntryPrice, "entry price above the minimum");
```

Comment:
this is super confusing. I know that that variable was repurposed already before, but would be nicer to have another local variable holding it. On the first glance is looks awefully wrong, althought it's not. Just because of the naming.

Reply:
The `sentValues.loanTokenSent` will be used by `_finalizeOpen` & `_emitOpeningEvents` after repurposed.
If we remove the "repurposed" approach and add new local variable holding it, meaning that we have to add new parameters in the `_finalizeOpen` & `_emitOpeningEvents`, which will require more gas.

Reply:
Will add new property in the struct